integration/e2e_test.go

```diff
@@ -580,6 +581,15 @@ func TestE2E_API_DeployEvents(t *testing.T) {
 
 	time.Sleep(1 * time.Second)
 
+	// perform `eth_call` to read the stored value
+	storedValue, err := rpcTester.call(contractAddress, callRetrieve)
+	require.NoError(t, err)
+	assert.Equal(
+		t,
+		"0000000000000000000000000000000000000000000000000000000000000539",
```

Comment:
either use the value from above that we store to create this encoded value (that would be best) or at least leave a comment saying what this value is.

Reply:
Updated in https://github.com/onflow/flow-evm-gateway/pull/124/commits/9acbc3100f7d80be75e1e1e1b48f9c7630aabb54

Reply:
This is basically the ABI-encoding of the value `1337`. I am not sure we have any helpers in the repo to produce the ABI-encoding value.